static/elements/chromedash-color-status.js

```diff
@@ -0,0 +1,29 @@
+import {LitElement, html} from 'https://unpkg.com/@polymer/lit-element@latest/lit-element.js?module';
+
+const CYAN = 120;
+const DEFAULT_MAX = 7;
+
+class ChromedashColorStatus extends LitElement {
+  static get properties() {
+    return {
+      max: {type: Number}, // From parent
```

Comment:
what's the `// From parent` comment for?

Reply:
I'm noting where the value is coming from. Here it means the max value is set in the attribute of the component, in the parent component template.

Reply:
Changed to "From attribute" to be consistent with others.